#diff -Naur linkchecker-4.5.orig/Makefile linkchecker-4.5/Makefile
#--- linkchecker-4.5.orig/Makefile	2006-09-24 12:16:10.000000000 +0200
#+++ linkchecker-4.5/Makefile	2006-10-21 02:54:16.000000000 +0200
#@@ -1,5 +1,5 @@
# # This Makefile is only used by developers.
#-PYVER := 2.4
#+PYVER := $(shell python -c 'from distutils import sysconfig; print sysconfig.get_python_version()')
# PYTHON := python$(PYVER)
# VERSION := $(shell $(PYTHON) setup.py --version)
# HOST=www.debian.org
diff -Naur linkchecker-4.5.orig/cgi-bin/lc.cgi linkchecker-4.5/cgi-bin/lc.cgi
--- linkchecker-4.5.orig/cgi-bin/lc.cgi	2006-06-23 21:07:51.000000000 +0200
+++ linkchecker-4.5/cgi-bin/lc.cgi	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2000-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/cgi-bin/lc.fcgi linkchecker-4.5/cgi-bin/lc.fcgi
--- linkchecker-4.5.orig/cgi-bin/lc.fcgi	2006-06-23 21:08:04.000000000 +0200
+++ linkchecker-4.5/cgi-bin/lc.fcgi	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2000-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/config/reindent.py linkchecker-4.5/config/reindent.py
--- linkchecker-4.5.orig/config/reindent.py	2006-05-03 19:57:15.000000000 +0200
+++ linkchecker-4.5/config/reindent.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#! /usr/bin/python2.4
+#! /usr/bin/python
 # -*- coding: iso-8859-1 -*-
 
 # Released to the public domain, by Tim Peters, 03 October 2000.
diff -Naur linkchecker-4.5.orig/doc/en/Makefile linkchecker-4.5/doc/en/Makefile
--- linkchecker-4.5.orig/doc/en/Makefile	2006-08-16 23:22:26.000000000 +0200
+++ linkchecker-4.5/doc/en/Makefile	2006-10-21 02:53:13.000000000 +0200
@@ -14,7 +14,7 @@
 
 .PHONY: nav
 nav: all
-	python2.4 ../bfknav.py
+	python ../bfknav.py
 
 .PHONY: homepage
 homepage: nav
diff -Naur linkchecker-4.5.orig/doc/rest2htmlnav linkchecker-4.5/doc/rest2htmlnav
--- linkchecker-4.5.orig/doc/rest2htmlnav	2006-07-23 01:25:24.000000000 +0200
+++ linkchecker-4.5/doc/rest2htmlnav	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.4
+#!/usr/bin/env python
 # -*- coding: iso-8859-1 -*-
 
 # Author: David Goodger
diff -Naur linkchecker-4.5.orig/install-rpm.sh linkchecker-4.5/install-rpm.sh
--- linkchecker-4.5.orig/install-rpm.sh	2006-07-26 10:27:17.000000000 +0200
+++ linkchecker-4.5/install-rpm.sh	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-python2.4 setup.py install --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+python setup.py install --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
 # 'brp-compress' compresses the manpages without distutils knowing.
 # The sed scripts append ".gz" resp. ".bz2" suffixes to the affected
 # manpage filenames.
diff -Naur linkchecker-4.5.orig/linkchecker linkchecker-4.5/linkchecker
--- linkchecker-4.5.orig/linkchecker	2006-09-21 14:43:53.000000000 +0200
+++ linkchecker-4.5/linkchecker	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2000-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/po/Makefile linkchecker-4.5/po/Makefile
--- linkchecker-4.5.orig/po/Makefile	2006-05-03 19:57:16.000000000 +0200
+++ linkchecker-4.5/po/Makefile	2006-10-21 02:53:13.000000000 +0200
@@ -2,7 +2,7 @@
 MSGFMT := msgfmt
 MSGMERGE := msgmerge
 POSOURCES = $(shell find ../linkcheck -name \*.py) \
-	../linkchecker /usr/lib/python2.4/optparse.py
+	../linkchecker /usr/lib/python$(shell python -c 'from distutils import sysconfig; print sysconfig.get_python_version()')/optparse.py
 LDIR = ../share/locale
 PACKAGE = linkchecker
 TEMPLATE = $(PACKAGE).pot
diff -Naur linkchecker-4.5.orig/scripts/get.py linkchecker-4.5/scripts/get.py
--- linkchecker-4.5.orig/scripts/get.py	2006-05-03 19:55:42.000000000 +0200
+++ linkchecker-4.5/scripts/get.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2004-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/scripts/head.py linkchecker-4.5/scripts/head.py
--- linkchecker-4.5.orig/scripts/head.py	2006-05-03 19:55:42.000000000 +0200
+++ linkchecker-4.5/scripts/head.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2004-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/scripts/parsefile.py linkchecker-4.5/scripts/parsefile.py
--- linkchecker-4.5.orig/scripts/parsefile.py	2006-05-03 19:55:42.000000000 +0200
+++ linkchecker-4.5/scripts/parsefile.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2004-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/setup.cfg linkchecker-4.5/setup.cfg
--- linkchecker-4.5.orig/setup.cfg	2006-07-26 10:08:08.000000000 +0200
+++ linkchecker-4.5/setup.cfg	2006-10-21 02:53:13.000000000 +0200
@@ -7,7 +7,7 @@
 provides = linkchecker
 group = Web/Utilities
 install_script = install-rpm.sh
-python = python2.4
+python = python
 
 [bdist_wininst]
 install-script = install-linkchecker.py
diff -Naur linkchecker-4.5.orig/setup.py linkchecker-4.5/setup.py
--- linkchecker-4.5.orig/setup.py	2006-09-22 12:15:34.000000000 +0200
+++ linkchecker-4.5/setup.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 # Copyright (C) 2000-2006 Bastian Kleineidam
 #
diff -Naur linkchecker-4.5.orig/test.py linkchecker-4.5/test.py
--- linkchecker-4.5.orig/test.py	2006-07-23 01:25:12.000000000 +0200
+++ linkchecker-4.5/test.py	2006-10-21 02:53:13.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 # -*- coding: iso-8859-1 -*-
 #
 # SchoolTool - common information systems platform for school administration
